Dockerfile.cockpit

```diff
@@ -1,6 +1,9 @@
 FROM fedora:latest
 MAINTAINER Xiaofeng Wang <xiaofwan@redhat.com>
 
+# to avoid the JS code doing checks we create this here
+RUN mkdir -p /tmp/failed-image
```

Comment:
Can you please be more specific why this needs to be done explicitly? It adds an entire extra docker layer and thus makes everything slower. docker's `-v` option already creates target directories in the container, so this shouldn't be necessary? If it is, then please `&&` it to an existing `RUN` to avoid the extra layer.

Reply:
Updated. Thanks.